web/templates/stream_settings/new_stream_user.hbs

```diff
@@ -1,4 +1,4 @@
-<tr>
+<tr class="subscriber_row" data-user-id="{{user_id}}" >
```

Comment:
`settings-subscriber-row` might be better?

Reply:
Updated the class name to `settings-subscriber-row`.